test/delay-api.test.js

```diff
@@ -0,0 +1,33 @@
+import t from 'tap'
+import Fastify from 'fastify'
+
+import slowDownPlugin from '../index.js'
+import { convertToMs } from '../lib/helpers.js'
+import { HEADERS, DEFAULT_OPTIONS } from '../lib/constants.js'
+import { slowDownAPI } from './helpers.js'
+
+t.test('Should start to delay the API', async t => {
+  const fastify = Fastify()
+  await fastify.register(slowDownPlugin)
+  t.teardown(() => {
+    fastify.close()
+  })
+
+  fastify.get('/', async () => 'Hello fastify-slow-down!')
+
+  await slowDownAPI(() => fastify.inject('/'))
```

Comment:
it would be better if this function also accepted the number of requests it has to do to the endpoint, without assuming that we're using the default value